src/System.Linq.Parallel/tests/QueryOperators/AverageTests.cs

```diff
@@ -77,6 +79,16 @@ public static void Average_Long_Longrunning(Labeled<ParallelQuery<int>> labeled,
         }
 
         [Theory]
+        [MemberData("Ranges", (object)(new int[] { 2 }), MemberType = typeof(UnorderedSources))]
+        public static void Average_Long_Overflow(Labeled<ParallelQuery<int>> labeled, int count)
```

Comment:
> nullable long is in there

Yeah, I'd written ```Int32 / Nullable<Int32>``` in my question, but I'd neglected to use the triple ticks and the generic qualifier got cut off.

> Can't overflow the addition for Int32, because of the loft to long, you'd run into the iteration limit first, so I didn't initially plan on including it. I can certainly add it, although it'd be outerloop (take a few minutes to run).

That's if we were trying to force the overflow to come from this particular checked area.  But the long will eventually be brought back to an int and an overflow will occur, right?  Shouldn't we have a test that verifies that happens (or do we already)?  That should be very quick.